.gitignore

```diff
@@ -71,3 +71,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+
+edx-ora2/
```

Comment:
@nasthagiri it looks like the 'make install' command from the wiki here: https://openedx.atlassian.net/wiki/display/TNL/ORA+FAQ#ORAFAQ-InitialSetup
creates this dir. I got Christina to thumb this change on https://github.com/edx/edx-ora2/pull/966